packages/core/src/App/Components/Layout/Header/dtrader-v2/__tests__/account-switcher-dtrader-v2.spec.tsx

```diff
@@ -0,0 +1,333 @@
+                        landing_company_name: Jurisdiction.MALTA_INVEST,
+                        balance: 10000,
+                    },
+                    CR90001397: {
+                        account_category: 'trading',
+                        account_type: 'binary',
+                        broker: BROKER_CODE.CR,
+                        created_at: 1718810862,
+                        currency: 'tUSDT',
+                        currency_type: CURRENCY_TYPE.CRYPTO,
+                        is_disabled: 0,
+                        is_virtual: 0,
+                        landing_company_shortcode: Jurisdiction.SVG,
+                        linked_to: [],
+                        token: 'mock_token',
+                        landing_company_name: Jurisdiction.SVG,
+                        balance: 0,
+                    },
+                    CR90001398: {
+                        account_category: 'trading',
+                        account_type: 'binary',
+                        broker: BROKER_CODE.CR,
+                        created_at: 1718810884,
+                        currency: 'BTC',
+                        currency_type: CURRENCY_TYPE.CRYPTO,
+                        is_disabled: 0,
+                        is_virtual: 0,
+                        landing_company_shortcode: Jurisdiction.SVG,
+                        linked_to: [],
+                        token: 'mock_token',
+                        landing_company_name: Jurisdiction.SVG,
+                        balance: 0,
+                    },
+                    CR90001399: {
+                        account_category: 'trading',
+                        account_type: 'binary',
+                        broker: BROKER_CODE.CR,
+                        created_at: 1718810892,
+                        currency: 'ETH',
+                        currency_type: CURRENCY_TYPE.CRYPTO,
+                        is_disabled: 0,
+                        is_virtual: 0,
+                        landing_company_shortcode: Jurisdiction.SVG,
+                        linked_to: [],
+                        token: 'mock_token',
+                        landing_company_name: Jurisdiction.SVG,
+                        balance: 0,
+                    },
+                    CR90001400: {
+                        account_category: 'trading',
+                        account_type: 'binary',
+                        broker: BROKER_CODE.CR,
+                        created_at: 1718810905,
+                        currency: 'LTC',
+                        currency_type: CURRENCY_TYPE.CRYPTO,
+                        is_disabled: 0,
+                        is_virtual: 0,
+                        landing_company_shortcode: Jurisdiction.SVG,
+                        linked_to: [],
+                        token: 'mock_token',
+                        landing_company_name: Jurisdiction.SVG,
+                        balance: 0,
+                    },
+                    CR90001401: {
```

Comment:
just a question :) Do we need the entire list of all these accounts for the tests to pass?

Reply:
Actually no, I can delete 4 fields from each object. It was a cope of object from my console, I just replace some values with the fake once